cg/apps/orderform/orderform_parser.py

```diff
@@ -29,7 +38,7 @@ def group_cases(self) -> Dict[str, List[OrderSample]]:
         LOG.info("Group samples under respective case")
         cases = {}
         for sample in self.samples:
-            case_id = sample.case_id
+            case_id = sample.family_name
```

Comment:
why remove case_id?

Reply:
since it was redundantly used to hold the family_name but it had the name we usually use for variables/arguments holding a family.internal_id string